jetbrains-rider/src/software/aws/toolkits/jetbrains/services/clouddebug/DotNetDebuggerSupport.kt

```diff
@@ -247,7 +191,7 @@ class DotNetDebuggerSupport : DebuggerSupport() {
 
         val debugArgs = StringBuilder()
             .append("RESHARPER_HOST_LOG_DIR=$remoteDebuggerLogPath ")
```

Comment:
nit: we could change it to a list then `.joinToString(' ')` then we wouldn't have to have the trailing spaces